### PR TITLE
Migrate to Cetmodules 3.13.00 and modern CMake

### PR DESCRIPTION
**Pull request by chissg**
* CMake modules FindPandoraSDK.cmake, FindPandoraMonitoring.cmake and
  FindLCContent.cmake supplement package-native generated config files
  providing:

  * Targets carrying library location, header location, and transitive
    library dependencies conferred via headers.

  * Setup of dependencies:

    PandoraPFA::LCContent -> PandoraPFA::PandoraMonitoring -> ROOT::Core
                          |                                |  ROOT::Graf3d
                         \|/                               /
                PandoraPFA::PandoraSDK <-------------------

* Minimize points of maintenance by maximizing commonality between
  Cetmodules-based and standalone builds:

  * Single project() call, specifying version once only.

  * Common library source list generation.

  * Common find_package() calls and option setting.

### CODE REVIEW

File: Modules/FindLCContent.cmake [hunk 1]
```diff
@@ -0,0 +1,53 @@
+### MIGRATE-NO-ACTION
```
Comment (AndyChappell) [inline Modules/FindLCContent.cmake:1]: I think this find module can be removed - LCContent is the linear collider equivalent to LArContent, so LArContent should never have a need to find this package.

**Comment by AndyChappell** [comment]
Thanks for this, Chris! Other than the minor comment against FindLCContent.cmake this looks mergeable to me.

**Comment by lgarren** [comment]
trigger build

**Comment by chissg** [comment]
For any MRB build with the new `larpandoracontent`, one now needs MRB >= `v6_04_02`

**Comment by lgarren** [comment]
mrb  v6_04_02 is now on larsoft cvmfs

**Comment by lgarren** [comment]
approve

**Pull request merged**